libs/sdk-ui-dashboard/src/model/commandHandlers/drill/resolveDrillToCustomUrl.ts

```diff
@@ -161,8 +161,8 @@ export function* loadAttributeElementsForDrillIntersection(
     return [...mappedElements, ...loadedElement];
 }
 
-const encodeParameterIfSet = (parameter: string | undefined): string | undefined =>
-    parameter === undefined ? parameter : encodeURIComponent(parameter);
+const encodeParameterIfSet = (parameter: string | undefined | null): string | undefined | null =>
+    parameter === undefined || parameter === null ? parameter : encodeURIComponent(parameter);
```

Comment:
Nitpick: `parameter === undefined || parameter === null` can be simplified to `isNil(parameter)` from lodash

Reply:
fixed